src/templates/globals/utils/FooterComment.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+
+const FooterComment = () => {
```

Comment:
Is it possible to have a general `Comment` component and pass in the comment as a child prop?

Reply:
good call as I'll be using this kind of comment in the head as well, will refactor